islandora_transforms/BPL_MODS_to_solr.xslt

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- BPL requires some indexing of cases not handled by standard MODS slurping -->
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:foxml="info:fedora/fedora-system:def/foxml#"
+  xmlns:mods="http://www.loc.gov/mods/v3"
+     exclude-result-prefixes="mods">
+
+  <xsl:template match="mods:mods[1]" mode="slurp_for_bpl">
+    <xsl:param name="prefix">bpl_mods_</xsl:param>
+    <xsl:param name="suffix">_ms</xsl:param>
+
+
+    <!--Non-date-typed regular name/namePart -->
+    <xsl:for-each select="mods:name[@type='personal']/mods:namePart[not(@type='date')]">
+      <field>
+        <xsl:attribute name="name">
+          <xsl:value-of select="concat($prefix, 'name_personal_namePart_not_date', $suffix)"/>
```

Comment:
... More what I was trying to get at, was to use templates instead of for-each stuff... Then it _would_ be expected to generate these types of field names... But yeah, super minor...